pkg/kubelet/kuberuntime/logs/logs.go

```diff
@@ -376,14 +377,16 @@ func ReadLogs(ctx context.Context, path, containerID string, opts *LogOptions, r
 			continue
 		}
 		// Write the log line into the stream.
-		if err := writer.write(msg); err != nil {
+		if err := writer.write(msg, isNewLine); err != nil {
 			if err == errMaximumWrite {
 				klog.V(2).Infof("Finish parsing log file %q, hit bytes limit %d(bytes)", path, opts.bytes)
 				return nil
 			}
 			klog.Errorf("Failed with err %v when writing log for log file %q: %+v", err, path, msg)
 			return err
 		}
+		/* The subsequent content is the rest part of current line or not*/
```

Comment:
please change this to:
`// Check if a new line character has been reached`

Reply:
The comment has been addressed.

Reply:
thanks.
please ping the #sig-node channel on slack if you need approval for the PR.
/ok-to-test